src/test/ui/traits/vtable/vtable-diamond.stderr

```diff
@@ -0,0 +1,56 @@
+error: Vtable Entries: [
+    MetadataDropInPlace,
```

Comment:
If `Method` were using `Instance`, it could just be a generic "function pointer" concept and could even be used to represent the "drop in place" entry - but I'm not sure how desirable that is.

Reply:
`Method` change is done. I'll leave changing "drop in place" to follow up work - #87253 .